backend/x86_dsl.mli

```diff
@@ -68,7 +68,7 @@ val mem64_rip: data_type -> ?ofs:int -> string -> arg
 module D : sig
   (** Directives *)
 
-  val align: int -> unit
+  val align: data:bool -> int -> unit
```

Comment:
Is `data` here referring to section name? In any case, it is unused in both x86_masm and x86_gas, can it be removed from the AST?

Reply:
No, it determines whether nop or zero padding is used.  @matthew4850 Please add a comment about this.